Hearthstone Deck Tracker/Config.cs

```diff
@@ -396,7 +396,10 @@ public class Config
 		[DefaultValue(-1)]
 		public int IgnoreNewsId = -1;
 
-		[DefaultValue(true)]
+        [DefaultValue(true)]
+        public bool ShowNewsBar = true;

```

Comment:
There is no need for a new config value, checking `IgnoreNewsId != -1` is sufficient.